examples/diff.c

```diff
@@ -335,3 +357,54 @@ static void diff_print_stats(git_diff *diff, struct opts *o)
 	git_buf_free(&b);
 	git_diff_stats_free(stats);
 }
+
+void die(char* message){
+	fprintf(stderr, message);
+	exit(EXIT_FAILURE);
+}
+
+struct pair extract_from_string(const char *cmd_opt)
+{
+	struct pair p;
+	char **endptr;
+
+
+	if (*cmd_opt=='/') {
+		p.first = DEFAULT_THRESHOLD;
+		cmd_opt++;
+		p.second = strtol(cmd_opt,endptr,'A');
+		if (**endptr!='%') 
```

Comment:
This itself should not be an error. It is possible to write `-B10/20` without git complaining about it.